src/settings.js

```diff
@@ -1,20 +1,20 @@
 export default {
   2018: {
     title: 'ISIC Challenge 2018',
-    covalicApiBase: 'https://challenge.kitware.com/api/v1/',
+    stadeApiBase: process.env.VUE_APP_STADE_API_BASE,
```

Comment:
Up to @brianhelba, but I think we'll want to start referring to this as challengeApiBase from now on.